current-branch-name/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Current Branch Name
+description: Sanitise branch name for use
```

Comment:
I wonder if this description might be expanded on a bit, maybe something like "Gets and sanitizes the current branch for branch, tag or pull request workflows"?

Reply:
I have expanded on this description